woodwork/tests/logical_types/test_logical_types.py

```diff
@@ -17,6 +19,7 @@
 from woodwork.tests.testing_utils.table_utils import to_pandas
 from woodwork.utils import import_or_none
 
+dd = import_or_none("dask.dataframe")
```

Comment:
Do we need this? Doesn't seem like it is used in the other new code that was added?